src/test/k6/QuizExerciseAPIs.js

```diff
@@ -77,7 +79,7 @@ export default function (data) {
     sleep(delay);
 
     group('Artemis Quiz Exercise Participation Websocket Stresstest', function () {
-        const userId = parseInt(__VU) + userOffset;
+        const userId = parseInt(__VU) + userOffset + userIdOffset;
```

Comment:
Why do we need userIdOffset in addition to userOffset? Is increasing the userOffset by 100 while running the k6 tests not enought?

Reply:
As far as i understand the useroffset i used to create a difference between the VUs while the userIdOffset is used to make sure that every VU uses an ID >= 100.